src/applications/SurvivalPredictor.cpp

```diff
@@ -542,9 +617,45 @@ VectorDouble SurvivalPredictor::SurvivalPredictionOnExistingModel(const std::str
 		logger.WriteError("Error in reading the file: " + modeldirectory + "/Survival_ZScore_Std.csv. Error code : " + std::string(e1.what()));
 		return results;
 	}
-	//----------------------------------------------------
+  MatrixType features6Matrix;
+  try
+  {
+    reader->SetFileName(modeldirectory + "/Survival_SelectedFeatures_6Months.csv");
+    reader->SetFieldDelimiterCharacter(',');
+    reader->HasColumnHeadersOff();
+    reader->HasRowHeadersOff();
+    reader->Parse();
+    features6Matrix = reader->GetArray2DDataObject()->GetMatrix();
+    selectedfeatures_6months.SetSize(features6Matrix.size());
+    for (unsigned int i = 0; i < features6Matrix.size(); i++)
+      selectedfeatures_6months[i] = features6Matrix(i, 0);
+  }
+  catch (const std::exception& e1)
+  {
+    logger.WriteError("Error in reading the file: " + modeldirectory + "/Survival_SelectedFeatures_6Months.csv. Error code : " + std::string(e1.what()));
+    return results;
+  }
+  MatrixType features18Matrix;
+  try
+  {
+    reader->SetFileName(modeldirectory + "/Survival_SelectedFeatures_18Months.csv");
+    reader->SetFieldDelimiterCharacter(',');
+    reader->HasColumnHeadersOff();
+    reader->HasRowHeadersOff();
+    reader->Parse();
+    features18Matrix = reader->GetArray2DDataObject()->GetMatrix();
+    selectedfeatures_18months.SetSize(features18Matrix.size());
+    for (unsigned int i = 0; i < features18Matrix.size(); i++)
+      selectedfeatures_18months[i] = features18Matrix(i, 0);
+  }
+  catch (const std::exception& e1)
+  {
+    logger.WriteError("Error in reading the file: " + modeldirectory + "/Survival_SelectedFeatures_18Months.csv. Error code : " + std::string(e1.what()));
+    return results;
+  }
+  //----------------------------------------------------
 	VariableSizeMatrixType FeaturesOfAllSubjects;
-	FeaturesOfAllSubjects.SetSize(qualifiedsubjects.size(), 161);
+	FeaturesOfAllSubjects.SetSize(qualifiedsubjects.size(), 164);
```

Comment:
what is 164?

Reply:
these number of features are fixed and not dependent on user input